tests/tasks/generators/clusterloader/load.yaml

```diff
@@ -73,16 +76,21 @@ spec:
     image: public.ecr.aws/kit/clusterloader2:0213bea
     onError: continue
     script: |
+      #!/bin/bash
```

Comment:
why do you need this ?

Reply:
`fork/exec /tekton/scripts/script-0-424g4: exec format error` ,In my case i got this error ,But after i added the she bang command ,It resolved ,By default its set -e

Reply:
I believe we need this here. If not adding shebang, tekton will prepend
```
#!/bin/bash
set -e
```
which will make the script exit when we get a non-zero exit code. It prevents us from collecting the CL2 exit code if it's not zero. Adding `#!/bin/bash` ourselves to ensure `set -e` is not added.